### PR TITLE
Typos in docs

### PR DESCRIPTION
**Pull request by smoe**
Just a few bits that I noted while reading the documentation myself.

### CODE REVIEW

File: docs/src/config/ini-homing.adoc [hunk 1]
```diff
@@ -6,10 +6,10 @@
 
 Homing sets the zero origin of the G53 machine coordinates. +
 Soft limits are defined relative to the machine origin. +
-The soft limits automatically decelerate and stop the axes before they hits the limits switches +
-A properly setup and functioning machine will not move beyond soft(ware) limits and +
+The soft limits automatically decelerate and stop the axes before they hit the limits switches +
+A properly configured and functioning machine will not move beyond soft(ware) limits and +
 will have the machine origin set as repeatable as the home switch/index mechanism is. +
-Linuxcnc can be  homed by eye (alignment marks), with switches, with switches and encoder index, or use absolute encoders. +
+Linuxcnc can be homed by eye (alignment marks), with switches, with switches and an encoder index, or by using absolute encoders. +
```
Comment (c-morley) [inline docs/src/config/ini-homing.adoc:12]: IMHO, the original 'by using' make more sense or something like this.
Linuxcnc can be homed:
    by eye
   with switches
   with switches and encoder index
  or with absolute encoders

**Comment by silopolis** [comment]
Le mar. 28 déc. 2021 à 04:38, c-morley ***@***.***> a écrit :

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In docs/src/config/ini-homing.adoc
> <https://github.com/LinuxCNC/linuxcnc/pull/1471#discussion_r775710993>:
>
> >  will have the machine origin set as repeatable as the home switch/index mechanism is. +
> -Linuxcnc can be  homed by eye (alignment marks), with switches, with switches and encoder index, or use absolute encoders. +
> +Linuxcnc can be homed by eye (alignment marks), with switches, with switches and an encoder index, or by using absolute encoders. +
>
> IMHO, the original 'by using' make more sense or something like this.
> Linuxcnc can be homed:
> by eye
> with switches
> with switches and encoder index
> or with absolute encoders
>

I second that.

>

File: docs/src/config/ini-homing.adoc (hunk 1 above)
Comment (smoe) [reply docs/src/config/ini-homing.adoc:12]: The "by using" is the correction, not the original. Your itemized presentation is also fine.  Please amend.

**Comment by silopolis** [comment]
Le mar. 28 déc. 2021 à 12:14, Steffen Möller ***@***.***> a
écrit :

> The "by using" is the correction, not the original. Your itemized
> presentation is also fine. Please amend.
>
Looks like xmas had worse consequences than expected 😅
Sorry folks

**Comment by c-morley** [comment]
My apologies. I mustta been tired :)

**Pull request merged**